src/backend/src/prisma/schema.prisma

```diff
@@ -114,20 +114,24 @@ model User {
   createdAssemblies         Assembly[]              @relation(name: "assemblyCreator")
   deletedMaterials          Material[]              @relation(name: "materialDeleter")
   createdMaterials          Material[]              @relation(name: "materialCreator")
+  Team                      Team[]
```

Comment:
can you rename this to archivedTeams, and add a name to the relation on the team and use that with @relation(name: "name") here